--- origsrc/enchant-1.6.1/configure.ac	2017-04-04 16:13:57.000000000 -0500
+++ src/enchant-1.6.1/configure.ac	2017-07-24 01:43:20.001552500 -0500
@@ -492,7 +492,9 @@ build_zemberek=no
 
 AC_ARG_ENABLE(zemberek, AS_HELP_STRING([--disable-zemberek],[enable the experimental zemberek (turkish) backend @<:@default=auto@:>@]), build_zemberek="$enableval", build_zemberek=no)
 
+if test "$build_zemberek" != no; then
 PKG_CHECK_MODULES(ZEMBEREK, [dbus-glib-1 >= 0.62], build_zemberek=yes, build_zemberek=no)
+fi
 
 AC_SUBST(ZEMBEREK_CFLAGS)
 AC_SUBST(ZEMBEREK_LIBS)
