--- origsrc/enchant-1.6.0/configure.in	2010-04-01 15:56:07.000000000 -0500
+++ src/enchant-1.6.0/configure.in	2015-10-27 21:50:37.269320100 -0500
@@ -48,8 +48,6 @@ have_cxx=yes
 AC_PROG_CXX(,have_cxx=no)
 AM_CONDITIONAL(WITH_CXX, test "x$have_cxx" = "xyes")
 
-AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
-
 PKG_CHECK_MODULES(ENCHANT, [glib-2.0 >= 2.6 gmodule-2.0])
 
 dnl ===========================================================================
